packages/trader/src/index.ts

```diff
@@ -54,9 +55,10 @@ export async function run(logger: winston.Logger, web3: Web3): Promise<void> {
       web3,
       gasEstimator,
       account: accounts[0],
-      dsProxyFactoryAddress: config.dsProxyFactoryAddress || getAddress("DSProxyFactory", networkId),
+      dsProxyFactoryAddress: config.dsProxyConfig?.dsProxyFactoryAddress || getAddress("DSProxyFactory", networkId),
       dsProxyFactoryAbi: getAbi("DSProxyFactory"),
-      dsProxyAbi: getAbi("DSProxy")
+      dsProxyAbi: getAbi("DSProxy"),
+      availableAccounts: config.dsProxyConfig?.availableAccounts || 1
```

Comment:
nit: just curious -- what are the downsides to just assume all accounts loaded into web3 are available to send transactions? Backwards compatibility?

Reply:
if the other EOAs dont have permission to access the DSProxy then transactions will fail. we could update the `DsProxyManager` to figure out how many EOAs it has loaded that have permissions then dynamically update this value at execution time? WDYT?

Reply:
Why would we load those EOAs into the bot if they didn't have access to the DSProxy?